packages/block-library/src/search/edit.js

```diff
@@ -53,6 +53,12 @@ const CSS_UNITS = [
 	{ value: 'px', label: 'px', default: PX_WIDTH_DEFAULT },
 ];
 
+// Used to calculate adjustment to inner button border radius.
+const DEFAULT_INPUT_PADDING = 4;
+
+// Minimum inner radius of 1px as having none when there is an outer radius jars.
+const MIN_INNER_RADIUS = 1;
+
 export default function SearchEdit( {
```

Comment:
Same here, it feels there are a lot of changes here where I'd have expected everything to be absorbed by the support flag and hooks?